arviz/tests/test_rcparams.py

```diff
@@ -35,6 +35,23 @@ def test_rc_context_file():
     assert rcParams["data.load"] == "lazy"
 
 
+def test_bad_rc_file():
+    """Test bad value raises error."""
+    path = os.path.dirname(os.path.abspath(__file__))
+    with pytest.raises(ValueError, match="Bad val "):
+        read_rcfile(path + "/bad.rcparams")
+
+
+def test_warning_rc_file(caplog):
+    """Test invalid lines and duplicated keys log warnings and bad value raises error."""
+    path = os.path.dirname(os.path.abspath(__file__))
+    read_rcfile(path + "/test.rcparams")
```

Comment:
```suggestion
    read_rcfile(os.path.join(path, "/test.rcparams"))
```

windows uses `\` as a separator, so we have to play this game, or use `pathlib`.

Reply:
Should the `/` in `"/test.rcparams"` be removed?

Reply:
Yes